x/finality/keeper/grpc_query.go

```diff
@@ -271,5 +271,22 @@ func (k Keeper) SigningInfos(ctx context.Context, req *types.QuerySigningInfosRe
 	if err != nil {
 		return nil, err
 	}
-	return &types.QuerySigningInfosResponse{FpSigningInfos: signInfos, Pagination: pageRes}, nil
+	return &types.QuerySigningInfosResponse{SigningInfos: k.convertToSigningInfosResponse(signInfos), Pagination: pageRes}, nil
+}
+
+func (k Keeper) convertToSigningInfoResponse(info types.FinalityProviderSigningInfo) types.SigningInfoResponse {
```

Comment:
suggestion to not attach functions to keeper if they do not use anything from the keeper 
```suggestion
func convertToSigningInfoResponse(info types.FinalityProviderSigningInfo) types.SigningInfoResponse {
```